src/app/features/notification/notification-rule/notification-condition/notification-condition-editor/notification-condition-editor.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, Inject, CUSTOM_ELEMENTS_SCHEMA } from "@angular/core";
+import { MatButtonModule } from "@angular/material/button";
+import {
+  MAT_DIALOG_DATA,
+  MatDialogModule,
+  MatDialogRef,
+} from "@angular/material/dialog";
+import { DialogCloseComponent } from "app/core/common-components/dialog-close/dialog-close.component";
+import { JsonEditorComponent } from "app/core/common-components/json-editor/json-editor.component";
+
+@Component({
+  selector: "app-notification-condition-editor",
+  standalone: true,
+  imports: [
+    JsonEditorComponent,
+    MatDialogModule,
+    DialogCloseComponent,
+    MatButtonModule,
+  ],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+  templateUrl: "./notification-condition-editor.component.html",
+  styleUrl: "./notification-condition-editor.component.scss",
+})
+export class NotificationConditionEditorComponent {
```

Comment:
Let's rename "JsonEditorDialogComponent" and move this to core. This can be reused for any kind of json editing

Reply:
Can you extract this into its separate branch directly based into "master", also :blush: ?

Reply:
Sure, I have updated the component name according to your suggestion. 
Also, I have created the new [PR](https://github.com/Aam-Digital/ndb-core/pull/2798) for this which is based on `master`.